services/app/src/hooks/useGameId.js

```diff
@@ -1,11 +1,24 @@
 import { useCallback } from 'react';
+import { nanoid } from 'nanoid';
+
+const ROOM_ID_LENGTH = 8;
 
 export const useGameId = () => {
   const saveGameId = useCallback((id) => {
     window.localStorage.setItem('lastGameId', id);
   }, []);
 
-  const getGameId = () => localStorage.getItem('lastGameId');
+  const generateGameId = () => nanoid(ROOM_ID_LENGTH);
+
+  const getGameId = () => {
+    const gameId = localStorage.getItem('lastGameId');
+    if (!gameId) {
+      const id = generateGameId();
```

Comment:
I'm a bit confused, is the `gameId` is created in the frontend or the backend, as it seems like the ids are generated in both places?

Also, the backend uses `uuid` package while frontend uses `nanoid`. Any reason to use both of them?